src/commandsAndMenu.tsx

```diff
@@ -732,6 +733,31 @@ export function addCommands(
     icon: openIcon.bindprops({ stylesheet: 'menuItem' })
   });
 
+  commands.addCommand(ContextCommandIDs.openFileFromDiff, {
+    label: trans.__('Open File'),
+    execute: async _ => {
+      const widget = app.contextMenuHitTest((node: HTMLElement) => {
+        const nodeId = node.dataset.id;
+        return nodeId && nodeId.substring(0, 4) === 'diff';
```

Comment:
Would you mind adding `git-` as prefix to `id` at line 499. I fear that testing for `diff` may not be sufficiently robust.

```suggestion
        return nodeId && nodeId.substring(0, 4) === 'git-diff';
```

Reply:
@fcollonval 
But the first 4 characters of the nodeId is just `diff`, I don't think changing it to 'git-diff' would work.

Reply:
@BoscoCHW you need indeed to add `git-` to line 499 where the id is defined for that suggestion to work (I can't make suggestion on unchanged lines :wink: )

Reply:
@fcollonval That makes sense. I was just afraid it might break other codes if they also relied on parsing the NodeId to do other things.

Reply:
It should not be the case - as this is something to avoid as much as possible. But thanks for raising the questions, better to discuss it than moving forward blindly.

Reply:
@fcollonval Should I move forward with the suggestion then?

Reply:
Yes please